Ejercicio 4

gonzafernan opened the pull request:
Ahí va otro

gonzafernan committed:
Ejercicio 4

Ahí va otro

gonzafernan committed:
Update ejercicios.h

ichmew commented:
Dudas en la línea 24:
El drand48 se usa porque utilizaste srand48?? No funciona el rand común??
Esos factores para qué son?? Pura estética??

GinoAvanzini commented:
Parece que drand48 y srand48 se usa igual que rand y srand común, solo que da valores decimales (float). Yo lo hice así con rand(), supongo que es lo mismo
`  srand (time(NULL));
  for(i=0;i<n;i++){
    u[i]=(double)rand()/(double)rand();
    printf("%lf ",u[i]);
  } printf("\n");`

FernandezGFG commented:
Uhhh eso hay que cambiarlo igual, pero es porque el srand48 va en el main.
Esta la explicación en los comentarios. El srand48 te da números aleatorios con coma. Los factores yo los puse para que sean un poco más random y con más coma. Lo vi en internet. Supuestamente también aumenta el rango.

GinoAvanzini committed:
Update ej4.c

A pedido del gonza agrego la forma en que hice lo del vector aleatorio, después decidimos cuál usar y comentamos/descomentamos

ichmew commented:
Me parece que el código del Gino es más apropiado, usamos el rand y srand común que ya vimos en el práctico anterior.

FernandezGFG commented:
Es la única duda??? Arreglo y mergeamos???

ichmew commented:
Dalee, mergeá con el printf comentado que creo que no lo vamos a usar.

gonzafernan committed:
Update ej4.c

FernandezGFG commented:
Okaaa

gonzafernan committed:
Merge branch 'master' into ejercicio-4
